src/Drush/Commands/DrushCommandsGeneratorBase.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+namespace Combawa\Drush\Commands;
+
+use DrupalCodeGenerator\Asset\AssetCollection;
+use DrupalCodeGenerator\Asset\RenderableInterface;
+use DrupalCodeGenerator\Asset\Resolver\ReplaceResolver;
+use DrupalCodeGenerator\Helper\Dumper\DryDumper;
+use DrupalCodeGenerator\Helper\Dumper\FileSystemDumper;
+use DrupalCodeGenerator\Helper\QuestionHelper;
+use DrupalCodeGenerator\Helper\Renderer\TwigRenderer;
+use DrupalCodeGenerator\InputOutput\IO;
+use DrupalCodeGenerator\Twig\TwigEnvironment;
+use DrupalCodeGenerator\Utils;
+use Drush\Commands\DrushCommands;
+use Drush\DrupalFinder\DrushDrupalFinder;
+use Drush\Log\Logger;
+use Psr\Container\ContainerInterface as DrushContainer;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Filesystem\Filesystem;
+use Twig\Loader\FilesystemLoader as TemplateLoader;
+
+/**
+ * Combawa generator commands helper.
+ */
+abstract class DrushCommandsGeneratorBase extends DrushCommands {
+
+  const TEMPLATES_PATH = __DIR__ . '/../../../templates';
+
+  /**
+   * The DrupalFinder utility.
+   *
+   * @var \Drush\DrupalFinder\DrushDrupalFinder
+   */
+  protected DrushDrupalFinder $drupalFinder;
+
+  /**
+   * Command constructor.
+   */
+  public function __construct(
+    protected readonly Filesystem $fileSystem,
+    protected readonly TwigRenderer $renderer,
+  ) {
+    parent::__construct();
+    $this->renderer->setLogger(new Logger($this->output()));
+    $this->renderer->registerTemplatePath(static::TEMPLATES_PATH);
+  }
+
+  /**
+   * Command creator.
+   */
+  public static function createEarly(DrushContainer $drush_container): static {
+    return new static(
+      new Filesystem(),
+      new TwigRenderer(new TwigEnvironment(new TemplateLoader())),
+    );
+  }
+
+  /**
+   * DrupalFinder utility getter.
+   *
+   * @return \Drush\DrupalFinder\DrushDrupalFinder
+   *   The DrupalFinder utility.
+   */
+  public function drupalFinder(): DrushDrupalFinder {
+    if (!isset($this->drupalFinder)) {
+      $this->drupalFinder = $this->processManager()->getDrupalFinder();
+    }
+    return $this->drupalFinder;
+  }
+
+  /**
+   * Run the generation process.
+   *
+   * @param array $options
+   *   The options array given to the command.
+   */
+  protected function generate(array $options): int {
```

Comment:
Maybe the name of the method isn't the best choice. The idea is to avoid replicating these lines in our two commands. We could introduce more methods but I'm not convinced it would be worth it.